eng/build.ps1

```diff
@@ -102,9 +102,8 @@ function Get-Help() {
 
 function Assert-InstalledDependency($dependencyName)
 {
-  Write-Host "Checking for $dependencyName..."
-  if (((Get-ItemProperty HKLM:\Software\Microsoft\Windows\CurrentVersion\Uninstall\*).DisplayName -Match
-        $dependencyName).Length -le 0)
+  $envPath = $Env:Path
```

Comment:
You should use the Get-Command cmdlet to try to find the dependency instead of doing a regex match on the PATH. Get-Command will resolve the path of the executable from the PATH.

Reply:
Looks like this was resolved.